src/renderer/widgets/WalletDetails/ui/components/ProxiesList.tsx

```diff
@@ -33,15 +33,15 @@ export const ProxiesList = ({ walletId, className, canCreateProxy = true }: Prop
         <FootnoteText className="flex-1 px-2 text-text-tertiary">{t('accountList.addressColumn')}</FootnoteText>
       </div>
 
-      <ul className="flex flex-col h-full px-5 divide-y divide-divider overflow-y-auto overflow-x-hidden">
+      <ul className="flex flex-col px-5 divide-y divide-divider overflow-y-auto overflow-x-hidden h-[inherit]">
```

Comment:
Changed it back. Wiht only one proxy in the list it was centered in the modal but since I removed `flex-1` from proxy conponent below I reverted to 'h-full' now